test/vm-customize

```diff
@@ -37,6 +37,7 @@ parser.add_argument('-r', '--run-command', action='append', dest="commandlist",
                                            default=[], help='Run command inside virtual machine')
 parser.add_argument('-s', '--script', action='append', dest="scriptlist",
                                       default=[], help='Run selected script inside virtual machine')
+parser.add_argument('-u', '--upload', action='append', dest="uploadlist", default=[], help='Upload files to destination dir separated by : EXAMPLE: -u file.txt:/var/lib')
```

Comment:
nitpick: can you add a newline? For example, "help" or "default" could start on the next line like with the other arguments. This seems a bit long.